frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -317,6 +317,7 @@ export const dashboardLogic = kea({
         refreshAllDashboardItems: async (_, breakpoint) => {
             await breakpoint(100)
             dashboardItemsModel.actions.refreshAllDashboardItems({})
+            setTimeout(() => actions.loadDashboardItems(), 1000)
```

Comment:
I couldn't figure out how to easily make this less async, ideally we'd refresh a dashboard on the frontend and the backend should be able to handle everything for it and just return a dashboard with the updated items. 

But since the way we do it right now is by sending out individual listeners on the frontend to refresh each item, I thought we should just work off of that and await all the items that returned before manually refreshing the dashboard to prevent potential async issues. But consolidating all the item calls inside of dashboardLogic was difficult, so a setTimeout will work for now

Reply:
@liyiy I just tried this by adding `import time;time.sleep(5)` on line 154 of api/insight.py (simulating a longer running query). Because of the short interval it doesn't give all the queries enough time to load, so this won't work unfortunately.

However, I just removed this line entirely and it seems to work correctly?

Reply:
Well I know definitely we needed to add the `true` to some of the actions.loadResults but I'm unsure if the dashboard refresh status itself will render correctly since it didn't change for me without doing a `loadDashboardItems` again

Reply:
Okay, feels like we should probably fix that then instead of reloading the entire dashboard.

Reply:
That was fixed in this PR, so our only issue is the dashboard isn't getting those refreshed items afterwards